components/datetime/src/fields/symbols.rs

```diff
@@ -416,3 +434,61 @@ impl From<DayPeriod> for FieldSymbol {
         Self::DayPeriod(input)
     }
 }
+
+#[derive(Debug, Eq, PartialEq, Clone, Copy)]
+#[cfg_attr(
+    feature = "provider_serde",
+    derive(serde::Serialize, serde::Deserialize)
+)]
+pub enum TimeZone {
+    LowerZ,
```

Comment:
Suggestion: The other similar enums describe more of what the underlying data represents, rather than what the UTS 35 symbol is. I wouldn't mind having the UTS 35 symbol documented along with the enum, but I think it would be better to have more descriptive enums.

e.g. here is the Day symbol:

```rust
pub enum Day {
    DayOfMonth,
    DayOfYear,
    DayOfWeekInMonth,
    ModifiedJulianDay,
}
```

Reply:
I spent quite some time trying to think about what to name the enums, because I used descriptive enums when I implemented Noon and Midnight for DayPeriods. 

Unfortunately, there aren't always descriptions that match the symbol, for example:
* `Z` switches between localized GMT format and ISO8601 formats depending on the length. 
* `V` switches between BCP-47 ID, IANA ID, Exemplar City, and Generic Location format. 

Rather than attempt to describe them, or mix and match between descriptive names or symbol names, I made the decision to use only symbol names for all of time zones.